src/sagas/repository.ts

```diff
@@ -514,40 +523,9 @@ export function* removeDatasetPolicyMember({ payload }: any): any {
     });
   } catch (err) {
     showNotification(err);
-  }
-}
-
-export function* addCustodianToDataset({ payload }: any): any {
```

Comment:
This add/remove custodian work seems to be dead.  In user management, it was replaced by the ADD_SNAPSHOT/DATASET_POLICY_MEMBER work.

Reply:
Yeah, I like having the more general purpose method